ci/pipelines/build-and-deploy.yml

```diff
@@ -135,9 +135,10 @@ jobs:
               apt-get install -y nodejs
               bundle install --without development
               bundle exec middleman build
-              cp -r . ../bundled/
+              cp -r build/* ../build
```

Comment:
why not `mv` as with what you did on `product-page`?

Reply:
Concourse creates the build directory at the start of this step, so mv fails.